docs/release/releases/release-1.9.md

```diff
@@ -0,0 +1,36 @@
+# Cluster API v1.9
+
+## Timeline
+
+The following table shows the preliminary dates for the `v1.9` release cycle.
+
+| **What**                                             | **Who**      | **When**                    | **Week** |
+|------------------------------------------------------|--------------|-----------------------------|----------|
+| Start of Release Cycle                               | Release Lead | Monday 19th August 2024     | week 1   |
+| Schedule finalized                                   | Release Lead | Friday 23rd August 2024     | week 1   |
+| Team finalized                                       | Release Lead | Friday 23rd August 2024     | week 1   |
+| *v1.7.x & v1.8.x released*                           | Release Lead | Tuesday 10th September 2024 | week 4   |
+| *v1.7.x & v1.8.x released*                           | Release Lead | Tuesday 8th October 2024    | week 8   |
+| *v1.7.x & v1.8.x released*                           | Release Lead | Tuesday 5th November 2024   | week 12  |
+| v1.9.0-beta.0 released                               | Release Lead | Tuesday 5th November 2024   | week 12  |
+| Communicate beta to providers                        | Release Lead | Tuesday 5th November 2024   | week 12  |
```

Comment:
Might be
```suggestion
| Communicate beta to providers                        | Comm Lead | Tuesday 5th November 2024   | week 12  |
```
Also, wdyt about adding an entry on W13 about communicating the upcoming code freeze to the community

Reply:
Maybe W12? During KubeCon might be not great

Reply:
Yes because there will be no community or release team meeting on W13. I will add it in W12.